evm/v0.5/testv2/Median.test.ts

```diff
@@ -64,37 +76,36 @@ contract('Median', () => {
       {
         name: 'overflowing numbers',
         responses: [
-          h.bigNum(
+          ethers.utils.bigNumberify(
```

Comment:
What about mapping with `ethers.utils.bigNumberify`?

Reply:
I decided to forgo mapping to keep the code as simple as possible, don't really see the value of it in the case of small arrays